program/shinyApp/R/C.R

```diff
@@ -248,23 +248,23 @@ guide <<- cicerone::Cicerone$
     el = "sidebar_help_tab",
     title = "This is the Sidebar",
     position = "right",
-    description = HTML("Always start here to give cOmicsArt tasks.<br>We will go through the elements step by step."),
+    description = HTML("Always start here to give cOmicsArt tasks.<br>We will go through the elements step by step.")
   )$
   step(
     el = "ImageSelectArea",
     position = "right",
     title = "Important parameters for the analysis",
-    description = HTML("Here we have 1 parameter to set.<br> Select 'Youtube Tutorial' from the dropdown menu.")
+    description = HTML("Here we have 1 parameter to set.<br>Select <strong>Youtube Tutorial<\strong> from the dropdown menu.")
```

Comment:
```suggestion
    description = HTML("Here we have 1 parameter to set.<br>Select <strong>Youtube Tutorial</strong> from the dropdown menu.")
```
I think this is why the test fails